app/assets/javascripts/code/values/session.js

```diff
@@ -9,3 +9,27 @@ export const formatSessionForList = session => ({
   timeframe: session.timeframe,
   shortTypes: session.shortTypes
 });
+
+export class Session {
+  constructor(session) {
+    this.session = session;
+  };
+
+  roundedAverage() {
+    return Math.round(this.session.average)
+  };
+
+  average() {
+    return this.session.average
+  };
+
+  selectedSensorUnit(selectedSensor) {
+    this.session.streams[selectedSensor]["unit_symbol"];
```

Comment:
I'm not sure this belongs to session. maybe there should be a similar class for streams???? how would you do it?

Reply:
Looks ok for now. When there will be too many things related to streams in here we could extract

Reply:
Also, the quotes are unnecessary

Reply:
I get an error when I remove them. Am I missing something? 
```ReferenceError: unit_symbol is not defined
at Session.selectedSensorUnit (session.js?d68e:27)```